src/registerToolbar.ts

```diff
@@ -40,13 +29,14 @@ export default function registerToolbar() {
         type: ToolbarItemType.State,
     });
 
-    ToolbarItemFactory.register(CopyRectangle, {
-        name: "copyRectangle",
-        tooltip: "Copy Rectangle",
-        icon: "fa-clone",
-        group: "canvas",
-        type: ToolbarItemType.State,
-    });
+    // This is buggy and doesn't fully work yet
```

Comment:
we should remove it then :D, we should not have commented out code

Reply:
Removed.